classes/class-uagb-front-assets.php

```diff
@@ -133,12 +133,12 @@ public function enqueue_asset_files() {
 			$this->post_assets->enqueue_scripts();
 		}
 
-		if ( wp_is_block_theme() ) {
+		if ( function_exists( 'wp_is_block_theme' ) && wp_is_block_theme() ) {
 			$this->load_assets_for_fse_theme();
 		}
 
 		/* Archive & 404 page compatibility */
-		if ( is_archive() || ( is_home() && ! wp_is_block_theme() ) || is_search() || is_404() ) {
+		if ( is_archive() || ( is_home() && ( ! function_exists( 'wp_is_block_theme' ) && ! wp_is_block_theme() ) ) || is_search() || is_404() ) {
```

Comment:
This might not work as intended, since if the function doesn't exist, it won't be able to check if `! wp_is_block_theme()`.

```suggestion
		if ( is_archive() || ( is_home() && ( ! function_exists( 'wp_is_block_theme' ) || ! wp_is_block_theme() ) ) || is_search() || is_404() ) {
```

Reply:
Done